gnat2goto/driver/tree_walk.adb

```diff
@@ -1315,24 +1318,28 @@ package body Tree_Walk is
                       "Identifier not in class type. Type definition failed.");
          return;
       end if;
-      if not Is_Type (E)
-      then
+
+      if not Is_Type (E) then
          Report_Unhandled_Node_Empty (N, "Do_Full_Type_Declaration",
                                       "identifier not a type");
          return;
       end if;
 
       --  If this is the full_type_declaration of a previous
-      --  incomplete_type_declaration then the Incomplete_View of the
-      --  declaration will be present and the
+      --  private_type_declaration or incomplete_type_declaration then the
```

Comment:
then the either it ??

Reply:
The following may be a better worded comment:
      --  If this is the full_type_declaration of a previous
      --  private_type_declaration or incomplete_type_declaration then
      --  either it will have a private declaration or an incomplete view 
      --  respectively and it will have been registered when its
      --  private or incomplete_type_declaration was processed.
      --  If neither the private declaration or incomplete view are
      --  present then the full_type_declaration has to be registered